2001-3000/LIT2050Nagara.xml

```diff
@@ -66,6 +67,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <relation name="saws:fallsWithin" active="LIT2050Nagara" passive="LIT1136Apopht"/>
          </listRelation>
          <div type="edition">
+           <note>
+             This division into textparts and incipits follow <ref type="mss" corresp="BLorient757"></ref>
```

Comment:
corresp = within the document, target = outside, here `@target`

Reply:
I think that's fine here: https://betamasaheft.eu/Guidelines/?id=references (References to other entities in BM)